matrix-sdk/build.gradle

```diff
@@ -100,7 +100,7 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp:2.2.0'
     compile 'io.pristine:libjingle:9690@aar'
 
-    compile(name: 'olm-sdk', ext: 'aar')
+    compile 'org.matrix:olm-sdk:olm-sdk@aar'
```

Comment:
Unfortunately, this still uses the embedded binary file libs/olm-sdk.aar (if you remove this file it does not work anymore)
Is it the expected behavior?

Reply:
Note: this is due to the lines in build.gradle
>        flatDir {
>            dir 'libs'
>        }